mlflow/deployments/cli.py

```diff
@@ -490,6 +490,6 @@ def validate_config_path(_ctx, _param, value):
     help="The number of workers.",
 )
 def start_server(config_path: str, host: str, port: str, workers: int):
-    from mlflow.gateway.runner import run_app
```

Comment:
This was broken...I'll add some test coverage

Reply:
Filed #10470

Reply:
Thanks! <3